Content.Server/Explosion/EntitySystems/TwoStagedGrenadeSystem.cs

```diff
@@ -0,0 +1,65 @@
+using Content.Server.Explosion.Components;
+using Content.Shared.Singularity.Components;
+using Robust.Shared.Timing;
+using Robust.Shared.Audio;
+using Content.Shared.Audio;
+using Content.Server.Audio;
+using Content.Server.Singularity.Components;
+
+namespace Content.Server.Explosion.EntitySystems;
+
+public sealed class TwoStagedGrenadeSystem : EntitySystem
+{
```

Comment:
Comments.

Reply:
Yeah, there are no comments, they are in component